ush/global_det/global_det_wave_plots_grid2obs_timeseries_WindDir.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+###############################################################################
+# Name of Script: global_det_wave_timeseries.sh
+# Developers: Deanna Spindler / Deanna.Spindler@noaa.gov
+#             Mallory Row / Mallory.Row@noaa.gov
```

Comment:
You can change my information to your information or add or name to the list of developers